db/migrate/20130126215239_create_users.rb

```diff
@@ -5,7 +5,7 @@ def change
       t.string :referral_code
       t.integer :referrer_id
 
-      t.timestamps
+      t.timestamps null: true
```

Comment:
i didn't love changing old migrations, but if the point is to get rid of warnings only, i suppose this works?